docusaurus.config.js

```diff
@@ -20,7 +20,7 @@ const config = {
           sidebarPath: require.resolve("./sidebars.js"),
           routeBasePath: "/",
           // Please change this to your repo.
-          editUrl: "https://github.com/weatherxm/docs",
+          editUrl: "https://github.com/WeatherXM/docs/blob/main",
```

Comment:
This path directs the user to the respective file in the repository but does not automatically put the user into an edit mode (presenting them with the ability to fork the repository). I can change the path to place the user in edit mode, if this is preferred.

Reply:
I think it's better to not get them in edit mode automatically. So as it is now is OK.

Reply:
I agree that we should leave it as is `https://github.com/weatherxm/docs`.